frontend/packages/console-app/src/components/nodes/NodeDetailsPage.tsx

```diff
@@ -1,40 +1,52 @@
 import * as React from 'react';
+import * as _ from 'lodash';
 import { navFactory } from '@console/internal/components/utils';
 import { PodsPage } from '@console/internal/components/pod';
 import { ResourceEventStream } from '@console/internal/components/events';
+import { NodeKind } from '@console/internal/module/k8s';
 import { DetailsPage } from '@console/internal/components/factory';
 import { nodeStatus } from '../../status/node';
 import NodeDetails from './NodeDetails';
 import NodeTerminal from './NodeTerminal';
 import { menuActions } from './menu-actions';
 import NodeDashboard from './node-dashboard/NodeDashboard';
 
-const { editYaml, events, pods } = navFactory;
+const NodeDetailsPage: React.FC<React.ComponentProps<typeof DetailsPage>> = (props) => {
+  const { editYaml, events, pods } = navFactory;
 
-const pages = [
-  {
-    href: '',
-    name: 'Overview',
-    component: NodeDashboard,
-  },
-  {
-    href: 'details',
-    name: 'Details',
-    component: NodeDetails,
-  },
-  editYaml(),
-  pods(({ obj }) => (
-    <PodsPage showTitle={false} fieldSelector={`spec.nodeName=${obj.metadata.name}`} />
-  )),
-  events(ResourceEventStream),
-  {
-    href: 'terminal',
-    name: 'Terminal',
-    component: NodeTerminal,
-  },
-];
-const NodeDetailsPage: React.FC<React.ComponentProps<typeof DetailsPage>> = (props) => (
-  <DetailsPage {...props} getResourceStatus={nodeStatus} menuActions={menuActions} pages={pages} />
-);
+  const pagesFor = React.useCallback(
+    (node: NodeKind) =>
+      _.compact([
+        {
+          href: '',
+          name: 'Overview',
+          component: NodeDashboard,
+        },
+        {
+          href: 'details',
+          name: 'Details',
+          component: NodeDetails,
+        },
+        editYaml(),
+        pods(({ obj }) => (
+          <PodsPage showTitle={false} fieldSelector={`spec.nodeName=${obj.metadata.name}`} />
+        )),
+        events(ResourceEventStream),
+        node?.metadata?.labels?.['node.openshift.io/os_id'] !== 'Windows'
```

Comment:
@spadgett it was recommended in the BZ

Reply:
Correct... 
We have `node.openshift.io/os_id=Windows` label attached to all the windows nodes that have joined the cluster

Reply:
Hi @jhadvig @spadgett sorry to offer conflicting information but [other components](https://github.com/openshift/oc/pull/509/files) have been using `corev1.LabelOSStable` to differentiate between Linux and Windows nodes. I think it is better to use this upstream label, rather than the OpenShift label.

Reply:
The expected value of `LabelOSStable` for Windows nodes `windows`

Reply:
@sebsoto @PratikMahajan would it be safe to check both labels?

Reply:
The corev1 label should cover all cases as its applied by the kubelet, but I dont see any current harm by doing checking for either label